src/components/DashboardCampaigns.tsx

```diff
@@ -10,46 +11,56 @@ interface DashboardCampaignProps {
     orgId: string;
 }
 
+const useStyles = makeStyles((theme) => ({
+    responsiveFlexBox: {
+        alignItems: 'center',
+        display: 'flex',
+        gap: '5%',
+        justifyContent: 'flex-end',
+        [theme.breakpoints.down('sm')]: {
+            flexDirection: 'column',
+        },
+    },
+}));
+
 const DashboardCampaigns = ({ orgId } : DashboardCampaignProps) : JSX.Element => {
     const campaignsQuery = useQuery(['campaigns', orgId], getCampaigns(orgId));
     const campaigns = campaignsQuery.data;
+    const classes = useStyles();
 
     return (
-        <View>
-            <Heading level={ 2 }>
+        <Box border={ 1 } m={ 2 } p={ 2 }>
+            <Typography component="h2" variant="body1">
                 <Msg id="pages.organize.currentProjects.title"/>
-            </Heading>
+            </Typography>
             { campaigns && (
-                <ul style={{ padding: 0 }}>
-                    { campaigns.map((item: ZetkinCampaign) => (
-                        <li key={ item.id } style={{
-                            border: '2px solid gray',
-                            listStyle: 'none',
-                            padding: '1rem',
-                        }}>
-                            <Link>
+                <Typography variant="h5">
+                    <List>
+                        { campaigns.map((item: ZetkinCampaign) => (
+                            <ListItem key={ item.id } style={{ background: grey[200], border: '1px solid', margin: '1rem 0'  }}>
                                 <NextLink href={ `/organize/${orgId}/campaigns/${item.id}` }>
-                                    <a>{ item.title }</a>
+                                    <Link color="inherit" href="#">
```

Comment:
It's not great that these links point to `#`. They still navigate to the correct path when clicked, probably because of DOM click event handlers on the containing `NextLink`, but it would be nice if the actual `a` element `href` attribute also had the right value, as that's shown by many browsers when hovering.

![image](https://user-images.githubusercontent.com/550212/121744472-40f3ec00-cb03-11eb-8725-9b05929d2b06.png)

Reply:
(This seems to be the case for a lot of links)